ElevenLabs/Packages/com.rest.elevenlabs/Runtime/TextToSpeech/TextToSpeechEndpoint.cs

```diff
@@ -256,6 +201,283 @@ void StreamCallback(Response partialResponse)
+        /// Possible values:<br/>
+        /// 0 - default mode (no latency optimizations)<br/>
+        /// 1 - normal latency optimizations (about 50% of possible latency improvement of option 3)<br/>
+        /// 2 - strong latency optimizations (about 75% of possible latency improvement of option 3)<br/>
+        /// 3 - max latency optimizations<br/>
+        /// 4 - max latency optimizations, but also with text normalizer turned off for even more latency savings
+        /// (best latency, but can mispronounce eg numbers and dates).
+        /// </param>
+        /// <param name="cancellationToken">Optional, <see cref="CancellationToken"/>.</param>
+        /// <returns>A tuple containing the <see cref="VoiceClip"/> and an array of <see cref="TimestampedTranscriptCharacter"/>.</returns>
+        public async Task<TranscribedVoiceClip> TextToSpeechWithTimestampsAsync(string text, Voice voice, VoiceSettings voiceSettings = null, Model model = null, OutputFormat outputFormat = OutputFormat.MP3_44100_128, int? optimizeStreamingLatency = null, CancellationToken cancellationToken = default)
+            => await TextToSpeechWithTimestampsAsync(new TextToSpeechRequest(voice, text, Encoding.UTF8, voiceSettings ?? voice.Settings ?? await client.VoicesEndpoint.GetDefaultVoiceSettingsAsync(cancellationToken), outputFormat, optimizeStreamingLatency, model), cancellationToken);
+
+        /// <summary>
+        /// Converts text into speech using a voice of your choice and returns audio.
+        /// </summary>
+        /// <param name="request"><see cref="TextToSpeechRequest"/>.</param>
+        /// <param name="cancellationToken">Optional, <see cref="CancellationToken"/>.</param>
+        /// <returns>A tuple containing the <see cref="VoiceClip"/> and an array of <see cref="TimestampedTranscriptCharacter"/>.</returns>
+        public async Task<TranscribedVoiceClip> TextToSpeechWithTimestampsAsync(TextToSpeechRequest request, CancellationToken cancellationToken = default)
+        {
+            var payload = JsonConvert.SerializeObject(request, ElevenLabsClient.JsonSerializationOptions);
+            var parameters = CreateRequestParameters(request);
+
+            var endpoint = $"/{request.Voice.Id}/with-timestamps";
+            
+            var response = await Rest.PostAsync(GetUrl(endpoint, parameters), payload, new RestParameters(client.DefaultRequestHeaders), cancellationToken);
+            response.Validate(EnableDebug);
+
+            if (!response.Headers.TryGetValue(HistoryItemId, out var clipId))
+            {
+                throw new ArgumentException("Failed to parse clip id!");
+            }
+
+            var responseJson = Encoding.UTF8.GetString(response.Data);
+            var jsonObject = JObject.Parse(responseJson);
+
+            string audioBase64 = jsonObject["audio_base64"].ToString();
+            byte[] audioBytes = Convert.FromBase64String(audioBase64);
```

Comment:
please use `var` on left hand side
```suggestion
            var audioBase64 = jsonObject["audio_base64"].ToString();
            var audioBytes = Convert.FromBase64String(audioBase64);
```